ci/travis/run-build.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+set -e
+
+TRAVIS_BUILD_DIR="${TRAVIS_BUILD_DIR:-.}"
+
+# cd to docker build dir if it exists
+if [ -d /docker_build_dir ] ; then
+	cd /docker_build_dir
+	TRAVIS_BUILD_DIR="/docker_build_dir"
+fi
+
+. ./ci/travis/lib.sh
+
+if [ -f "${TRAVIS_BUILD_DIR}/env" ] ; then
+	echo_blue "Loading environment variables"
+	cat "${TRAVIS_BUILD_DIR}/env"
+	. "${TRAVIS_BUILD_DIR}/env"
+fi
+
+KCFLAGS="-Werror"
+export KCFLAGS
+
+APT_LIST="build-essential bc u-boot-tools flex bison libssl-dev"
+
+if [ "$ARCH" == "arm64" ] ; then
+	APT_LIST="$APT_LIST gcc-aarch64-linux-gnu"
+else
+	APT_LIST="$APT_LIST gcc-arm-linux-gnueabihf"
+fi
+
+adjust_kcflags_against_gcc() {
+	# FIXME: remove this function once kernel gets upgrade and
+	#        these have been fixed upstream. Currently, these are
+	#	 for kernel 4.19
+	GCC="${CROSS_COMPILE}gcc"
+	if [ "$($GCC -dumpversion | cut -d. -f1)" -ge "8" ]; then
+		KCFLAGS="$KCFLAGS -Wno-error=stringop-truncation"
```

Comment:
with respect to these warnings I would like to add that we've been using an older base of rpi-4.19.y, so it's likely that some of them were fixed in upstream 4.19.y or some of them were fixed in the rpi-4.19.y; 

i would probably need to investigate which are still valid and which can be removed; i could just remove all of them and just let `-Werror`